src/scene/export.ts

```diff
@@ -228,3 +263,17 @@ export const getExportSize = (
 
   return [width, height];
 };
+
+export const serializeToSvg = (
+  elements: readonly NonDeletedExcalidrawElement[],
+  // FIXME: extract a type for this, or delete exportToSvg and inline?
```

Comment:
This type does seem to appear in multiple placfes, so probably worth explicitly declaring it.